src/litserve/loops.py

```diff
@@ -399,63 +400,176 @@ class _BaseLoop(ABC):
       - unbatch
       - encode_response
 
+    To implement a custom loop, subclass this class and implement the `run` method. The `run` method should execute the
+    hooks in the desired order.
+
+    `__call__` method is the entry point for the worker process. It calls the `run` method in a loop until the worker is
+    terminated.
+
+    Example:
+
+    ```python
```

Comment:
```suggestion
    .. code-block:: python
```

Reply:
markdowns are easier to read. And this is not using Sphinx

Reply:
yes but is this anywhere used like for generating docs?